data.js

```diff
@@ -3905,5 +3905,19 @@ module.exports = [
       "Web Development",
       "Beginner",
     ],
+  },
+  {
+    name: "Making operations easy using GitHub CLI",
+    description: "All about learning GitHub CLI, helping tool for developers.",
+    url: "https://tauqeerahmad5201.medium.com/making-things-easy-using-github-cli-b5624e9b1350",
```

Comment:
The url should point to your entire blog, not just a single post. Please update it to `https://tauqeerahmad5201.medium.com`